android/app/build.gradle

```diff
@@ -1,5 +1,6 @@
 apply plugin: "com.android.application"
 apply plugin: 'kotlin-android'
+apply plugin: 'spoon'
```

Comment:
Hi @enriquegh 

This really is an awesome feature. I only have 1 challenge. It works on RDC and Sauce Emulators, but not on my local emulator. I get `Unable to create output dir:` errors.
I've checked some pages, but I can't the real solution. It looks like it's being blocked by the permission modal. 
Do you have a specific configuration for your emulator?

Reply:
Fixed in dff1a49